tools/find_untranslated_strings.py

```diff
@@ -0,0 +1,159 @@
+#! /usr/bin/env python
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+from collections import namedtuple
+
+import re
+
+TransAndKey = namedtuple("TransAndKey", "translation, key")
```

Comment:
Ah, ok, leave TransAndKey as is, but translation is a global and must be renamed to TRANSLATION.